src/main/cli/settings.c

```diff
@@ -1701,6 +1706,12 @@ const clivalue_t valueTable[] = {
     { "display_name",     VAR_UINT8  | MASTER_VALUE | MODE_STRING, .config.string = { 1, MAX_NAME_LENGTH, STRING_FLAGS_NONE }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, displayName) },
 #endif
     { "model_id",         VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 99 }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelId) },
+    { "model_param1_type",  VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_PARAM_TYPE }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam1Type) },
+    { "model_param1_value", VAR_INT16  | MASTER_VALUE, .config.minmaxUnsigned = { -32000, 32000 }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam1Value) },
+    { "model_param2_type",  VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_PARAM_TYPE }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam2Type) },
+    { "model_param2_value", VAR_INT16  | MASTER_VALUE, .config.minmaxUnsigned = { -32000, 32000 }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam2Value) },
+    { "model_param3_type",  VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_PARAM_TYPE }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam3Type) },
+    { "model_param3_value", VAR_INT16  | MASTER_VALUE, .config.minmaxUnsigned = { -32000, 32000 }, PG_PILOT_CONFIG, offsetof(pilotConfig_t, modelParam3Value) },
```

Comment:
INT16 is not unsigned...
Also, why limit it to -32000...32000 if the data type is not known

Reply:
Thanks! Fixed that.